openjdk-8.yaml

```
@@ -0,0 +1,222 @@
+package:
+  name: openjdk-8
+  version: 8u372-b07
+  epoch: 0
+  description: "IcedTea distribution of OpenJDK 8"
+  copyright:
+    - license: GPL-2.0-or-later
+  dependencies:
+    runtime:
+      - openjdk-8-jre
+      - java-cacerts
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - build-base
+      - bash
+      - findutils
+      - zip
+      - file
+      - gawk
+      - util-linux-dev
+      - libxslt-dev
+      - autoconf
+      - automake
+      - libtool
+      - linux-headers
+      - coreutils
+      - ca-certificates
+      - freetype-dev
+      - cups-dev
+      - libx11-dev
+      - libxext-dev
+      - libxrender-dev
+      - libxrandr-dev
+      - libxtst-dev
+      - libice-dev
+      - libxt-dev
+      - alsa-lib-dev
+      - libffi-dev
+      - krb5-dev
+      - pkgconf-dev
+      - attr-dev
+      - expat-dev
+      - fontconfig-dev
+      - libxi-dev
+      - libjpeg-dev
+      - giflib-dev
+      - lcms2-dev
+      - java-gcj-compat-default-jvm
```

This should probably be openjdk-7-default-jvm.

```suggestion
      - openjdk-7-default-jvm
```